app/Support/NftImageUrl.php

```diff
@@ -55,8 +55,9 @@ private static function getAlchemyCdn(string $url, ?ImageSize $imageSize): ?stri
 
             $size = 'w_'.$imageSize->width().',h_'.$imageSize->height();
             $newPath = preg_replace('/(?<=\/)upload(?=\/)/', "upload/{$size}", $path);
+            $scaledPath = preg_replace('/thumbnailv2/', 'scaled', $newPath);
```

Comment:
can we update existing image urls by replacing it as such as well? if so, it would be good to add a migration that would update the existing NFTs so they are all using the new format